pkg/util/slice/slice.go

```diff
@@ -57,5 +57,43 @@ func (p Int64Slice) Len() int           { return len(p) }
 func (p Int64Slice) Less(i, j int) bool { return p[i] < p[j] }
 func (p Int64Slice) Swap(i, j int)      { p[i], p[j] = p[j], p[i] }
 
-// Sorts []int64 in increasing order
+// SortInts64 sorts []int64 in increasing order
 func SortInts64(a []int64) { sort.Sort(Int64Slice(a)) }
+
+// IntSlicesEqual returns whether the two slices contain the same ints.
+func IntSlicesEqual(x, y []int) bool {
```

Comment:
I don't think I can justify this PR for 1.3 if it contains IntSlicesEqual and StringSlicesEqual:
* it's not under cloudprovider/aws. 
* sorting (i.e. changing) the args is a little questionable, and should at least be mentioned in the doc

I think it's easier to just use a sets.String (see pkg/util/sets/string.go)